demo/src/pages/demos/pulltorefresh.tsx

```diff
@@ -1,258 +1,126 @@
+
   useEffect(() => {
     const contentWrapper = document.getElementById("pull-contentWrapper");
     const content = document.getElementById("pull-content");
     const prepend = document.getElementById("prepend");
     const append = document.getElementById("append");
 
-    const getInfo = (pos) => {
-      const state =
-        pos > 0 ? (pos - axes.axis.scroll.range[1]) / 100 : -pos / 100;
-      return {
-        isAdd: state > 0.8,
-        isTop: pos < 0,
-      };
-    };
-
-    const getMaxRange = () => {
-      return (
-        content.getBoundingClientRect().height -
-        contentWrapper.getBoundingClientRect().height
-      );
-    };
-
-    const getItem = () => {
-      const el = document.createElement("li");
-      el.className = "pull_drw addblinking";
-      el.innerHTML = `<div class="pull_im">
-          <img src="../../../static/img/demos/pulltorefresh/${Math.floor(Math.random() * 50 + 1)}.jpg" width="110"/>
-        </div>
-        <div class="pull_tx">
-          <span class="pull_tit">egjs is Javascript components group that brings easiest and fastest way to build a web application in your way</span>
-        </div>`;
-      return el;
-    };
-
     // 1. Initialize eg.Axes
-    const axes = new Axes({
-      scroll: {
-        range: [0, getMaxRange()],
-        bounce: 100,
-      },
-    });
+    axes.axis.scroll.range = [0, getMaxRange(content, contentWrapper)];
+
+    // 2. Initialize inputTypes and connect it
+    axes.connect(
+      ["", "scroll"],
+      new PanInput(contentWrapper, {
+        scale: [0, -1],
+      })
+    );
 
-    // 2. attach event handler
+    // 3. attach change event handler
     axes.on({
       change: ({ pos }) => {
         content.style[Axes.TRANSFORM] = `translate3d(0, ${-pos.scroll}px, 0)`;
         if (axes.isBounceArea()) {
           const info = getInfo(pos.scroll);
           if (info.isAdd) {
-            info.isTop
-              ? (prepend.innerText = "Release to prepend")
-              : (append.innerText = "Release to append");
+            if (info.isTop) {
+              prepend.innerText = "Release to prepend";
+            } else {
+              append.innerText = "Release to append";
+            }
           } else {
-            info.isTop
-              ? (prepend.innerText = "Pull to prepend")
-              : (append.innerText = "Pull to append");
+            if (info.isTop) {
+              prepend.innerText = "Pull to prepend";
+            } else {
+              append.innerText = "Pull to append";
+            }
           }
         }
       },
+    });
+  }, []);
+
+  useEffect(() => {
+    const contentWrapper = document.getElementById("pull-contentWrapper");
+    const content = document.getElementById("pull-content");
+
+    // 4. attach release event handler on each slides update
+    axes.off("release");
+    axes.on({
       release: ({ depaPos }) => {
+        console.log("hi");
```

Comment:
remove plz.

Reply:
It seems that the code for testing was not properly caught in the eslint settings. Thanks for reporting.